public/theme/boltcms-2021/showcases.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'partials/_master.twig' %}
+
+{% block main %}
+    <body class="loading showcase">
+    <main>
+        <div data-scroll>
+            <div class="subheader">
+                <h1>{{ contenttype.name }}</h1>
+                <p>A fine selection of websites made with Bolt</p>
+            </div>
+            <div class="content">
+                {% for showcase in showcases %}
```

Comment:
Might be something to confirm with @Dsduit, but I think the design is meant to have 9 "featured" showcases, after which we have the `honorary mention`. If that's so, let's cap these at 8 here 🙂

Reply:
done